apps/crm/crm/src/app/crm/pages/events/events.component.html

```diff
@@ -38,6 +41,14 @@
     <ng-template colRef="isSecret" label="Hidden on Marketplace" let-isSecret sort>
       {{isSecret}}
     </ng-template>
+    <ng-template colRef label="Action" let-event sticky>
+      <a mat-icon-button [routerLink]="goToEdit(event)">
```

Comment:
```suggestion
      <a mat-icon-button [routerLink]=['/c/o/dashboard/crm/event', event.id]">
```

and remove `goToEdit`  method